docs/documentation/functions/aria-suite/aria-operations-networks/New-AriaNetworksNsxtDataSource.md

```diff
@@ -0,0 +1,177 @@
+# New-AriaNetworksNsxtDataSource
+
+## Synopsis
+
+Add a new NSX-T Data Source to VMware Aria Operations for Networks.
+
+## Syntax
+
+### credentials (Default)
+
+```powershell
+New-AriaNetworksNsxtDataSource -fqdn <String> [-username <String>] [-password <String>] -nickname <String>
+ -collectorId <String> -enabled <String> [<CommonParameters>]
+```
+
+### certificate
+
+```powershell
+New-AriaNetworksNsxtDataSource -fqdn <String> [-certificate <String>] [-privatekey <String>] -nickname <String>
+ -collectorId <String> -enabled <String> [<CommonParameters>]
+```
+
+## Description
+
+The `New-AriaNetworksNsxtDataSource` cmdlet allows a user to add in a NSX-T Manager as a new data source in VMware Aria Operations for Networks in order to collect network flow data.

```

Comment:
```suggestion
The `New-AriaNetworksNsxtDataSource` cmdlet allows a user to add an NSX-T Manager as a new data source in VMware Aria Operations for Networks in order to collect network flow data.

```

Reply:
```suggestion
The `New-AriaNetworksNsxtDataSource` cmdlet allows a user to add an NSX Manager as a new data source in VMware Aria Operations for Networks in order to collect network flow data.

```

Reply:
Resolved in https://github.com/vmware/power-validated-solutions-for-cloud-foundation/commit/954c74bc51b1fef11dc26b704cc60d52dd1bba80